sigstore-gradle/sigstore-gradle-sign-base-plugin/src/main/kotlin/dev/sigstore/sign/SigstoreSignExtension.kt

```diff
@@ -39,7 +39,7 @@ abstract class SigstoreSignExtension(private val project: Project) {
     abstract val sigstoreJavaVersion : Property<String>
 
     init {
-        sigstoreJavaVersion.convention("0.2.0")
+        sigstoreJavaVersion.convention("0.3.0")
```

Comment:
Looks like this should be "before 0.3.0 release"

Reply:
I'm not sure I understand. Also, I'm not sure I understand these test error messages.

Reply:
I think the change of return type from String to byte array was not really backward compatible: https://github.com/sigstore/sigstore-java/commit/f3b363d2f0c2c947c894c3ecbafbfc0f3ab4a463#diff-13a2e26b591d0577e3c7a3f31beedf30cf2645b61d4179ce14407d7157279ff6

Reply:
hah yeah I get that. But I don't think I understand why CI is failing but my local isn't.

Reply:
oh yeah, there's a separate config for CI.

Reply:
Do you mean `CI=Y` environment var?
Should the test print something like "Running reduced set of tests to get faster feedback, consider adding CI=Y to execute all the tests" ?